pagopa/io-app
# chore: [IOBP-333] Add sorting for pagoPA payment notices to barcodes choice screen

## Short description
This PR implements the sorting for the detected barcodes in case of multiple barcodes in the choice screen.

| Before | After |
| --- | --- |
|  <img src="https://github.com/pagopa/io-app/assets/6160324/2f13e766-5dbd-4856-8b9a-6df0c844ca34" width="200" /> |  <img src="https://github.com/pagopa/io-app/assets/6160324/8c2d2724-4da4-4f6e-9bb0-4d771bf98328" width="200" /> |

## List of changes proposed in this pull request
- [ts/features/walletV3/payment/screens/WalletPaymentBarcodeChoiceScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-333-in-caso-di-piu-codici-pago-pa-rilevati-ordinarli-per-importo-decrescente?expand=1#diff-df9889d94972386628b359910ce7aa1909a30d8a8d95b7931cf68fdc46499483): added sort function for the barcodes array

## How to test
From the scan screen, upload a file containing 3 or more barcodes and check that the results are ordered from the highest amount.
